moshirudderadapter/gradle.properties

```diff
@@ -0,0 +1,6 @@
+POM_ARTIFACT_ID=moshirudderadapter
+POM_PACKAGING=jar
+VERSION_CODE=1
+VERSION_NAME=0.0.1
```

Comment:
version seems wrong

Reply:
I was thinking to test the complete flow with these values, then update them.